service/src/main/kotlin/fi/espoo/evaka/invoicing/data/FeeDecisionQueries.kt

```diff
@@ -56,6 +56,16 @@ val feeDecisionQueryBase =
 
 val feeDecisionDetailedQueryBase =
     """
+    WITH finance_decision_handler_by_youngest_child AS (
+        SELECT
+            fee_decision_part.fee_decision_id AS decision_id,
+            finance_decision_handler.first_name AS first_name,
+            finance_decision_handler.last_name AS last_name,
+            row_number() OVER (PARTITION BY (fee_decision_id) ORDER BY date_of_birth DESC) AS rownum
+        FROM fee_decision_part
+        LEFT JOIN daycare ON fee_decision_part.placement_unit = daycare.id
+        LEFT JOIN employee AS finance_decision_handler ON finance_decision_handler.id = daycare.finance_decision_handler
+    )
```

Comment:
Decisions in draft state don't need to display this information, so just left join the `employee` table straight to the `fee_decision` table.